.github/storypoints.py

```diff
@@ -0,0 +1,148 @@
+            repos[repo['name']] = repo['id']
+
+        while 'next' in response.links:
+            response = self._call(response.links['next']['url'])
+            for repo in response.json():
+                repos[repo['name']] = repo['id']
+
+        return repos
+
+    def get_repo_issues(self, owner, repo, params=None):
+        url = '%s/repos/%s/%s/issues' % (self.endpoint, owner, repo)
+        print(params)
+        if params is None:
+            params = {}
+        if 'state' not in params:
+            params['state'] = 'all'
+        print(params)
+        response = self._call(url, params=params)
+        print(response.url)
+        issues = response.json()
+        while 'next' in response.links:
+            print('following "next" link %s' % response.links['next'])
+            response = self._call(response.links['next']['url'])
+            issues.extend(response.json())
+
+        return issues
+
+    def get_repo_commit_activity(self, owner, repo):
+        url = '%s/repos/%s/%s/stats/commit_activity' % (self.endpoint, owner, repo)
+        response = self._call(url)
+        if response.status_code == 202:
+            print('waiting for GitHub to generate statistics for %s...' % repo)
+            sleep(3)
+            return self.get_repo_commit_activity(owner, repo)
+        elif response.status_code == 200:
+            print(response.json())
+            return response.json()
+
+    def get_releases(self, owner, repo):
+        url = '%s/repos/%s/%s/releases' % (self.endpoint, owner, repo)
+        response = self._call(url)
+        if response.status_code == requests.codes.ok:
+            return response.json()
+
+    def get_tags(self, owner, repo):
+        url = '%s/repos/%s/%s/tags' % (self.endpoint, owner, repo)
+        response = self._call(url)
+        if response.status_code == requests.codes.ok:
+            return response.json()
+
+    def get_commit(self, owner, repo, sha):
+        url = '%s/repos/%s/%s/commits/%s' % (self.endpoint, owner, repo, sha)
+        response = self._call(url)
+        if response.status_code == requests.codes.ok:
+            return response.json()
+
+
+if __name__ == '__main__':
+    ghub = GitHubAPI(os.environ['GITHUB_USERNAME'], os.environ['GITHUB_AUTH_TOKEN'])
+    # get dict of repo by name -> id
+    repos = ghub.get_org_repos('Princeton-CDH')
+    print(repos)
+
+    issues = ghub.get_repo_issues('Princeton-CDH', 'mep-django')
+    print(len(issues))
+
+    zhub = ZenHubAPI(os.environ['ZENHUB_API_TOKEN'])
+
+    project = 'ppa'
+    repo_id = repos['ppa-django']
+    # project = 'mep'
+    # repo_id = repos['mep-django']
+    with open('issue_storypoints.csv', 'w') as csvfile:
+        # title, # images, # complete, findingaid url, plum url
+        fieldnames = ['story', 'estimate', 'project', 'url']
+        writer = csv.DictWriter(csvfile, fieldnames=fieldnames)
+        writer.writeheader()
+
+        print('%d total issues' % len(issues))
```

Comment:
Should we get rid of print statements? We probably don't want them anymore, and unlikely to be useful in the github action output, I think...

Reply:
added to todo